OptimizelySDKCore/OptimizelySDKCoreTests/OPTLYNotificationCenterTest2.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  OPTLYNotificationCenterTest2.swift
+//  OptimizelySDKCore
+//
```

Comment:
This needs to be the Optimizely copyright header.

Reply:
Copyright updated